resolwe/flow/executors/__init__.py

```diff
@@ -97,9 +103,10 @@ def run(self, data_id, script, verbosity=1):
 
         self.data_id = data_id
 
+        data_dir = settings.FLOW_EXECUTOR['DATA_DIR']
         dir_mode = getattr(settings, 'FLOW_EXECUTOR', {}).get('DATA_DIR_MODE', 0o755)
```

Comment:
We should add default settings and then not care if they are define or not. [This](https://docs.djangoproject.com/en/1.9/topics/settings/#custom-default-settings) may be interesting, if it is possible to define per-app defaults. Otherwise [DRF](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/settings.py) has such implementation, but it looks a bit complicated to me.

@kostko do you have any suggestion?